openssl/src/dh.rs

```diff
@@ -106,9 +197,50 @@ impl Dh<Params> {
     }
 }
 
+impl<T> Dh<T>
+where
+    T: HasPublic,
+{
+    /// Gets the public key from the DH instance.
+    ///
+    /// This corresponds to [`DH_get0_key`].
+    ///
+    /// [`DH_get0_key`]: https://www.openssl.org/docs/man1.1.0/crypto/DH_get0_key.html
+    pub fn get_public_key(&self) -> &BigNumRef {
```

Comment:
Rust conventions avoid the `get_` prefix, so this would just be called `public_key`.